dbt/selectors.yml

```diff
@@ -1,15 +1,15 @@
 selectors:
-  - name: qc_tests
-    description: Selector for running QC tests on iasWorld tables
+  - name: data_test_iasworld
```

Comment:
**nitpick (non-blocking):** I find it pretty confusing to have the selector named the same thing as the tag. Maybe we can prefix it with something?

Reply:
Great point, I renamed the selector to `select_data_test_iasworld` in b61a761a39614c77d8c853e5b0dc7ef1b7de27e0.